### ISSUE TITLE
Do we have a prometheus dashboard for this plugin or can we use any existing one?

### ISSUE THREAD

**Issue by yangou** [vvanholl/elasticsearch-prometheus-exporter#299]

**Comment by jamshid** [comment]
Yeah wondering the same thing, is there a recommended dashboard for this (standard?) prometheus exporter plugin or a list of ones that work with it?
I use this one https://grafana.com/grafana/dashboards/266 looks like it's by same author as this plugin. But it hasn't been updated in four years? 
I see a lot of "No data" in the dashboard with Elasticsearch 7.5.2, like the Document Index Rate panel.

**Comment by TheRealDwright** [comment]
I've submitted a PR with a dashboard I put together that uses this projects cluster metrics. This would have saved me a lot of time so I hope that it saves some time for somebody else!
